frontend/src/views/Transactions/Transactions.jsx

```diff
@@ -0,0 +1,185 @@
+import BCAlert from '@/components/BCAlert'
+import BCButton from '@/components/BCButton'
+import { DownloadButton } from '@/components/DownloadButton'
+import { ROUTES } from '@/constants/routes'
+import { useApiService } from '@/services/useApiService'
+import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Box, Typography } from '@mui/material'
+import { AgGridReact } from 'ag-grid-react'
+import * as dayjs from 'dayjs'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { dummy } from './data'
+import { getStatus } from '@/utils/getStatus'
+import { getOrgnization } from '@/utils/getOrganization'
+
+export const Transactions = () => {
+  const navigate = useNavigate()
+  const apiService = useApiService()
+
+  const [isDownloadingTransactions, setIsDownloadingTransactions] =
+    useState(false)
+  const [alertMessage, setAlertMessage] = useState('')
+  const [alertSeverity, setAlertSeverity] = useState('info')
+
+  const handleDownloadTransactions = async () => {
+    alert('Download Transactions clicked')
```

Comment:
Are there any dependencies to not implement this now?
If yes, then can you add a TODO comment please.

Reply:
Added TODO comments